docs/intro.rst

```diff
@@ -40,19 +40,39 @@ this order:
 
 To force a particular API, set the configuration variable ``qt_api`` in your ``pytest.ini`` file to
 ``pyqt5``, ``pyside``, ``pyqt4`` or ``pyqt4v2``. ``pyqt4v2`` sets the ``PyQt4``
-API to `version 2 <version2>`_.
+API to `version 2`_.
 
 .. code-block:: ini
 
     [pytest]
     qt_api=pyqt5
 
-
 Alternatively, you can set the ``PYTEST_QT_API`` environment
 variable to the same values described above (the environment variable wins over the configuration
 if both are set).
 
-.. _version2: http://pyqt.sourceforge.net/Docs/PyQt4/incompatible_apis.html
+Starting with ``pytest-qt`` version 2.0, ``PyQt`` or ``PySide`` are loaded after
```

Comment:
Probably this is too much information to be into the "introduction" part of the docs. Perhaps we can add an entry "A note about pyqt4v2", similar to [A note about QApplication.exit()](http://pytest-qt.readthedocs.io/en/latest/app_exit.html)?